### ISSUE TITLE
App no longer builds (master branch)

### ISSUE THREAD

**Issue by TheArcaneBrony** [th-ch/youtube-music#928]
Trying to install dependencies (`npm i`) results in the following error:
```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: youtube-music@1.18.0
npm ERR! Found: electron@20.1.1
npm ERR! node_modules/electron
npm ERR!   dev electron@"^20.1.1" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer electron@"^10.0.0 || ^11.0.0 || ^12.0.0 || ^13.0.0 || ^14.0.0 || ^15.0.0 || ^16.0.0 || ^17.0.0 || ^18.0.0" from custom-electron-prompt@1.5.0
npm ERR! node_modules/custom-electron-prompt
npm ERR!   custom-electron-prompt@"^1.5.0" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR! See /home/Rory/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/Rory/.npm/_logs/2022-12-24T13_02_04_502Z-debug-0.log
```

Adding `--force` causes a ENOENT to be thrown by npm.

Will try updating packages and report back if i have any luck!

**Comment by Zo-Bro-23** [comment]
What's your node version? (Run `node -v`)

**Comment by th-ch** [comment]
Hey, it seems to run fine here with node 14 ([node 16 is used in CI](https://github.com/th-ch/youtube-music/blob/335d515e22e4fe721ad73b919871c5656e7bd4c5/.github/workflows/build.yml#L21)) and yarn (not npm) - dependencies can be installed with `yarn` instead of `npm i`! Can you try and see if it works for you?

**Comment by Zo-Bro-23** [comment]
Also, `rimraf` seems to cause issues for me sometimes. Try editing `package.json` (lines 78-82) and removing the `yarn run clean && ` parts from the build scripts.

**Comment by JellyBrick** [comment]
Fixed at #1235
